### ISSUE TITLE
Set definition in code

### ISSUE THREAD

**Issue by electricwig** [coppolaemilio/dialogic#304]
Apologies if this is already a feature - but was wondering if there was some way to set a Dialogic 'definition' in code or link it to an existing variable? 
For example, I already have a singleton 'GameManager' with variables in it - such as GameManager.protagonist = "Name" - which can be set in my game at the start. But I would really like to then make or modify a dialogic definition (via on ready code along the lines of: "DialogicResources.progagonist = GameManager.protagonist" ) so that when a new dialogue starts up and the  definition [protagonist] is called in the dialogue box, it says whatever the player entered at the start of the game. 
I see there is a way to modify definitions in game, but I am afraid I can't work out how to set them to previously existing variables outside of Dialogic. 
And again, apologies if this has already been explained somewhere but I couldn't find it in the documentation or videos!

**Comment by coppolaemilio** [comment]
Hey! I saw you also sent me a message on reddit. Did you managed to find the `Dialogic.get_variable` and `Dialogic.set_variable` functions?

**Comment by electricwig** [comment]
Hi! I closed the issue when I found the get_variable and set_variable functions, but sent you the message when I couldn't seem to get them to work correctly .... 

So far in the Dialogic panel I have created a new definition called Protagonist and just set it to ProtagName for now.

Then in the _ready(): of my main game script I am trying to update it in code so that I can dynamically change the player name from an input screen later (just testing it for now), with this line of code: 

Dialogic.set_variable("ProtagName", "Graham")

But when I run the game, it still switches back to ProtagName and in the output it says: 

[Dialogic] saved definitions not present, creating file: 0

I even tried testing it using get_variable and in ready that _is_ indeed returning my new name (Graham), but again, when it actually hits the Dialogic cut scene it just goes back to the default name (ProtagName).
